src/gr11review/part1/Review2.java

```diff
@@ -1 +1,42 @@
 package gr11review.part1;
+import java.io.*;
+
+/**
+* Using a switch statement, create 4 joke options and a default error message
+@author Benjamin Teh
+*/
+
+public class Review2{
+  public static void main(String[] args) throws IOException{
+    BufferedReader keyboard = new BufferedReader(new InputStreamReader(System.in));
+    int numJoke;
+    
+    System.out.println("0 - print a joke about your hair");
+    System.out.println("1 - print a joke about your feet");
+    System.out.println("2 - print a joke about your clothes");
+    System.out.println("3 - print a joke about your teacher");
+    System.out.println("Choose a menu option: ");
```

Comment:
prompts should be `System.out.print` instead of `System.out.println` so that input can be on the same line as the prompt.